morituri/command/cd.py

```diff
@@ -279,15 +295,20 @@ def add_arguments(self):
         self.parser.add_argument('--track-template',
             action="store", dest="track_template",
             default=DEFAULT_TRACK_TEMPLATE,
-            help="template for track file naming (default default)")
+            help="template for track file naming")
         self.parser.add_argument('--disc-template',
             action="store", dest="disc_template",
             default=DEFAULT_DISC_TEMPLATE,
-            help="template for disc file naming (default default)")
+            help="template for disc file naming")
         self.parser.add_argument('-U', '--unknown',
             action="store_true", dest="unknown",
             help="whether to continue ripping if the CD is unknown",
             default=False)
+        self.parser.add_argument('--allow-freedb',
```

Comment:
How about the short option '-F' for **F**reeDB **F**allback?

Reply:
I'd rather not make using this option easy and didn't want to reserve a 1-letter option for a somehow "legacy" mode.

Reply:
All options should have a short form.

Reply:
Many don't (`--cdr`, `--track-template`, `--disc-template` in this module)

Reply:
It's something we're planning on fixing shortly, although I understand that you may not be privy to transient information on IRC.

Reply:
I also can't argue there's a consensus on this topic, I just strongly prefer BSD-style interfaces; so do what you want.